Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [evcc/evcc](https://hosted.weblate.org/projects/evcc/evcc/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/evcc/evcc/horizontal-auto.svg)

Xells1984 reviewed (approved):
Yesterday Translation Session

zigad committed:
Translated using Weblate (Slovenian)

Currently translated at 100.0% (1017 of 1017 strings)

Co-authored-by: Žiga Deisinger <ziga@deisinger.si>
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/sl/
Translation: evcc/evcc

RTTTC committed:
Translated using Weblate (Lithuanian)

Currently translated at 100.0% (1017 of 1017 strings)

Translated using Weblate (Lithuanian)

Currently translated at 100.0% (1017 of 1017 strings)

Co-authored-by: RTTTC <94727758+RTTTC@users.noreply.github.com>
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/lt/
Translation: evcc/evcc

Miodowa8 committed:
Translated using Weblate (Polish)

Currently translated at 66.7% (677 of 1014 strings)

Translated using Weblate (Polish)

Currently translated at 66.6% (676 of 1014 strings)

Co-authored-by: Dominik Trzebiński <trzebinik@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/pl/
Translation: evcc/evcc

Commit:
Translated using Weblate (Ukrainian)

Currently translated at 100.0% (1017 of 1017 strings)

Translated using Weblate (Ukrainian)

Currently translated at 100.0% (1017 of 1017 strings)

Co-authored-by: Максим Горпиніч <gorpinicmaksim0@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/uk/
Translation: evcc/evcc

amtssp committed:
Translated using Weblate (Danish)

Currently translated at 100.0% (1017 of 1017 strings)

Translated using Weblate (Danish)

Currently translated at 100.0% (1017 of 1017 strings)

Co-authored-by: amtssp <amtssp@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/da/
Translation: evcc/evcc

JonasTuff committed:
Translated using Weblate (Swedish)

Currently translated at 99.8% (1015 of 1017 strings)

Translated using Weblate (Swedish)

Currently translated at 99.9% (1016 of 1017 strings)

Translated using Weblate (Swedish)

Currently translated at 99.8% (1015 of 1017 strings)

Co-authored-by: Jonas Gate <tidy.koala8317@brot.se>
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/sv/
Translation: evcc/evcc

Xells1984 committed:
Translated using Weblate (Spanish)

Currently translated at 79.6% (810 of 1017 strings)

Translated using Weblate (French)

Currently translated at 100.0% (1017 of 1017 strings)

Translated using Weblate (Finnish)

Currently translated at 100.0% (1017 of 1017 strings)

Translated using Weblate (Dutch)

Currently translated at 100.0% (1017 of 1017 strings)

Translated using Weblate (Portuguese)

Currently translated at 100.0% (1017 of 1017 strings)

Co-authored-by: André W. <xell@users.noreply.hosted.weblate.org>
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/es/
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/fi/
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/fr/
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/nl/
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/pt/
Translation: evcc/evcc

Xells1984 committed:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 65.7% (669 of 1017 strings)

Translated using Weblate (Norwegian Bokmål)

Currently translated at 65.6% (668 of 1017 strings)

Translated using Weblate (Norwegian Bokmål)

Currently translated at 38.3% (390 of 1017 strings)

Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 75.4% (767 of 1017 strings)

Translated using Weblate (Turkish)

Currently translated at 100.0% (1017 of 1017 strings)

Translated using Weblate (Russian)

Currently translated at 100.0% (1017 of 1017 strings)

Translated using Weblate (Russian)

Currently translated at 99.7% (1014 of 1017 strings)

Translated using Weblate (Russian)

Currently translated at 80.8% (822 of 1017 strings)

Translated using Weblate (Russian)

Currently translated at 80.7% (821 of 1017 strings)

Translated using Weblate (Russian)

Currently translated at 45.8% (466 of 1017 strings)

Translated using Weblate (Russian)

Currently translated at 23.6% (241 of 1017 strings)

Translated using Weblate (Russian)

Currently translated at 23.5% (240 of 1017 strings)

Translated using Weblate (Croatian)

Currently translated at 100.0% (1017 of 1017 strings)

Translated using Weblate (Russian)

Currently translated at 18.4% (188 of 1017 strings)

Translated using Weblate (Czech)

Currently translated at 100.0% (1017 of 1017 strings)

Translated using Weblate (Bulgarian)

Currently translated at 100.0% (1017 of 1017 strings)

Translated using Weblate (Croatian)

Currently translated at 98.8% (1005 of 1017 strings)

Translated using Weblate (Bulgarian)

Currently translated at 87.1% (886 of 1017 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (1017 of 1017 strings)

Translated using Weblate (Italian)

Currently translated at 100.0% (1017 of 1017 strings)

Translated using Weblate (Portuguese)

Currently translated at 100.0% (1017 of 1017 strings)

Translated using Weblate (Italian)

Currently translated at 86.5% (880 of 1017 strings)

Translated using Weblate (Greek)

Currently translated at 100.0% (1017 of 1017 strings)

Translated using Weblate (Slovenian)

Currently translated at 99.9% (1016 of 1017 strings)

Translated using Weblate (Hungarian)

Currently translated at 100.0% (1017 of 1017 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (1017 of 1017 strings)

Translated using Weblate (Hungarian)

Currently translated at 100.0% (1017 of 1017 strings)

Translated using Weblate (Polish)

Currently translated at 100.0% (1017 of 1017 strings)

Translated using Weblate (Polish)

Currently translated at 82.0% (834 of 1017 strings)

Translated using Weblate (Greek)

Currently translated at 91.4% (930 of 1017 strings)

Translated using Weblate (Swedish)

Currently translated at 100.0% (1017 of 1017 strings)

Translated using Weblate (Dutch)

Currently translated at 100.0% (1017 of 1017 strings)

Translated using Weblate (French)

Currently translated at 100.0% (1017 of 1017 strings)

Translated using Weblate (Polish)

Currently translated at 67.2% (684 of 1017 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (1017 of 1017 strings)

Co-authored-by: André Westphal <xells1984@users.noreply.hosted.weblate.org>
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/bg/
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/cs/
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/el/
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/es/
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/fr/
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/hr/
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/hu/
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/it/
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/nb_NO/
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/nl/
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/pl/
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/pt/
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/ru/
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/sl/
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/sv/
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/tr/
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/zh_Hans/
Translation: evcc/evcc

WoBoW4U committed:
Translated using Weblate (Dutch)

Currently translated at 100.0% (1017 of 1017 strings)

Co-authored-by: WoBoW4U <w.b.oudshoorn@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/nl/
Translation: evcc/evcc

jonilehtola committed:
Translated using Weblate (Finnish)

Currently translated at 100.0% (1017 of 1017 strings)

Co-authored-by: Joni Lehtola <joni.lehtola+github@iki.fi>
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/fi/
Translation: evcc/evcc

BravoNLD committed:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 85.4% (869 of 1017 strings)

Translated using Weblate (Norwegian Bokmål)

Currently translated at 85.4% (869 of 1017 strings)

Co-authored-by: BravoNLD <14bravo@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/nb_NO/
Translation: evcc/evcc

Xells1984 committed:
Translated using Weblate (Romanian)

Currently translated at 100.0% (1017 of 1017 strings)

Translated using Weblate (Romanian)

Currently translated at 41.3% (421 of 1017 strings)

Translated using Weblate (Romanian)

Currently translated at 39.3% (400 of 1017 strings)

Translated using Weblate (Arabic)

Currently translated at 3.7% (38 of 1017 strings)

Translated using Weblate (Bulgarian)

Currently translated at 100.0% (1017 of 1017 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (1017 of 1017 strings)

Translated using Weblate (Norwegian Bokmål)

Currently translated at 100.0% (1017 of 1017 strings)

Translated using Weblate (Russian)

Currently translated at 100.0% (1017 of 1017 strings)

Translated using Weblate (Norwegian Bokmål)

Currently translated at 85.4% (869 of 1017 strings)

Co-authored-by: André Westphal <xells1984@users.noreply.hosted.weblate.org>
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/ar/
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/bg/
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/es/
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/nb_NO/
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/ro/
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/ru/
Translation: evcc/evcc

Commit:
Translated using Weblate (Luxembourgish)

Currently translated at 100.0% (1017 of 1017 strings)

Co-authored-by: Filipe Lima <filipe.limadacunha@outlook.com>
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/lb/
Translation: evcc/evcc

ftonellotto committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (1017 of 1017 strings)

Co-authored-by: ftonellotto <filippo.tonellotto@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/it/
Translation: evcc/evcc

dingoduffel committed:
Translated using Weblate (Dutch)

Currently translated at 100.0% (1017 of 1017 strings)

Co-authored-by: dingoduffel <dirk@katalise.be>
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/nl/
Translation: evcc/evcc

Xells1984 committed:
Translated using Weblate (Dutch)

Currently translated at 99.9% (1016 of 1017 strings)

Co-authored-by: André Westphal <xells1984@users.noreply.hosted.weblate.org>
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/nl/
Translation: evcc/evcc

weblate committed:
Merge remote-tracking branch 'origin/master'
